qtodotxt/ui/controllers/tasks_list_controller.py

```diff
@@ -194,8 +195,24 @@ def _copySelectedTasks(self):
             app = QtWidgets.QApplication.instance()
             app.clipboard().setText(text)
 
-    def availabilityEditAction(self):
+    def updateActions(self):
         if len(self.view.getSelectedTasks()) == 1:
-            self.editTaskAction.setDisabled(False)
+            self.enableTaskActions()
```

Comment:
but now you disable all actions immediately when you select multiple rows.
As I understand there should be a separate condition for editing tasks